packages/exampleReactNative/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "exampleReactNative",
+  "version": "0.0.1",
+  "private": true,
+  "workspaces": {
+    "nohoist": [
+      "react-native",
+      "react-native/**",
+      "react-native-randombytes",
+      "react-native-randombytes/**",
+      "@react-native-community/**",
+      "react-native-svg",
+      "react-native-svg/**"
+    ]
+  },
+  "scripts": {
+    "copy-workspace-packages": "yarn copy-identicon && yarn copy-ui-shared",
```

Comment:
Call this from the root packager script - so it just gets done without additional commands. (As a side note, having to do this is really horrible... but understand why, doesn't mean either of us need to like it, since it is "hacky")

Reply:
Do you mean to remove the `copy-workspace-packages` script and do `cd packages/exampleReactNative && yarn copy-identicon && yarn copy-ui-shared` in the root package.json scripts?